### PR TITLE
Feature branch setup

### PR DESCRIPTION
**Pull request by karthykarthick**
In this one i implemented everything as per the requirements

   - Initialize a git repo

   - Create a Heroku app

   - Commit and deploy to Heroku

   - Make sure the Heroku app works correctly

   - Push everything to Github

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,5 +34,18 @@
...
+  },
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.11.0",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.20.6",
+    "eslint-plugin-react-hooks": "^2.5.1",
+    "stylelint": "^13.3.3",
```
Comment (addod19) [inline package.json:48]: Kindly add big.js and prop-types to package.json file

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @karthykarthick, congratulations on this project :partying_face: Awesome :+1: App created with create-react-app :1st_place_medal: PR and README file professional :fireworks: Live link working correctly :slightly_smiling_face: However, let us kindly address the requirements that ae not met:
- Kindly add big.js and prop-types to our package.json file
- Kindly remove unnecessary files or assets like the logo.svg, reportWebVitals, or one of the CSS file

Once these are fixed, kindly submit another code review request :smile: 
_Hapy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @karthykarthick :hand: 
Nice job so far but we need to make some changes before moving on.

- [ ] We need to update the README file by adding descriptive information about the project. Right now the README lacks relevant information about the project since it still has the placeholder text generated by `create-react-app`.
Screenshot from 2021-02-03 13-37-02

- [ ] All code from the index.js file was deleted. Our app should at least start using React. Your index file should have at least:
```
import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';

ReactDOM.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
  document.getElementById('root')
);

```

- [ ] We also need to have the App.js file. The component should be as simple as this.

```JS
import React from 'react'

const App = () => {
  return (
    <h1>Hello World</h1>
  )
}
```

Happy coding :computer: 
**Sérgio Torres**

**Comment by bellom** [comment]
> ### Changes required 💪
> Hi @karthykarthick ✋
> Nice job so far but we need to make some changes before moving on.
> 
> * [ ]  We need to update the README file by adding descriptive information about the project. Right now the README lacks relevant information about the project since it still has the placeholder text generated by `create-react-app`.
>   Screenshot from 2021-02-03 13-37-02
> * [x]  All code from the index.js file was deleted. Our app should at least start using React. Your index file should have at least:
> 
> ```
> import React from 'react';
> import ReactDOM from 'react-dom';
> import App from './App';
> 
> ReactDOM.render(
>   <React.StrictMode>
>     <App />
>   </React.StrictMode>,
>   document.getElementById('root')
> );
> ```
> 
> * [x]  We also need to have the App.js file. The component should be as simple as this.
> 
> ```js
> import React from 'react'
> 
> const App = () => {
>   return (
>     <h1>Hello World</h1>
>   )
> }
> ```
> 
> Happy coding 💻
> **Sérgio Torres**

@karthykarthick Kindly make the changes mentions in this comment by @Torres-ssf 

Just like your heroku link, which display `hello`, running your app locally doesn't show that.

In other word; you need to add those necessary files.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @karthykarthick   :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

# Status: Changes Required! ♻️ 

**Kindly check out my quote comment above**


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

## Congratulations @karthykarthick :tada: 

### This project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:

**Comment(s)**
- For projects divided into milestones, always push your code to production whenever you're done with a particular feature/milestone.

Again, congratulations, you can now merge to development.
Do not forget to fill the project completion form.

**_Good luck with your next project._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**